GameEngine/Engine.cs

```diff
@@ -88,5 +86,27 @@ public static void Start(Func<IGameData> gameDataFactory, int loadFromSlot = 0)
                 }
             }
         }
+
+        public void StartNewGame()
+        {
+            string intro = gameData.GetGameIntroduction();
+            Console.Clear();
+            Console.WriteLine(intro);
+
+            Console.WriteLine($"First, we are going to set up your Character.");
+            Console.WriteLine($"Now, you are {gameData.PlayerName}, do you want to change it? Yes/No");
+
+            if(Console.ReadLine().Equals("yes", StringComparison.OrdinalIgnoreCase))
+            {
+                string PlayersNewName;
+                Console.Write($"You are changing your name, what is your new name?: ");
+                PlayersNewName = Console.ReadLine();
+                gameData.PlayerName = PlayersNewName;
```

Comment:
Can you make it so the IGameData interface only allows get; and not set; if that is possible ? I think we shouldn't store what the player's chosen name is here, but instead store it in the engine for now.